crates/pallets/starknet/src/tests/invoke_tx.rs

```diff
@@ -617,3 +620,285 @@ fn storage_changes_should_revert_on_transaction_revert() {
         assert_eq!(balance_value, vec![Felt252Wrapper::ZERO])
     })
 }
+
+#[test]
+fn given_hardcoded_contract_run_invoke_v1_without_fees_token_it_fails() {
```

Comment:
this test covers something already covered by "given_hardcoded_contract_run_invoke_v1_without_eth_with_strk_fee_token_it_fails".

The only way it would behave differently would be the thing making it fail with the first one were having strk, and not having strk would be the thing making it succeed.
It doesn't make a lot of sense, so you can just remove this one

Reply:
Ok i just wanted to be complete in my test, i totally agree